### ISSUE TITLE
Q: link to the hosted version

### ISSUE THREAD

**Issue by Bekt** [looplab/logspout-logstash#36]
Hello, this adapter is great! I'm maintaining a hosted version of this at [Bekt/logspout-logstash](https://github.com/Bekt/logspout-logstash) on Docker Hub. I was wondering if you could link to it somewhere in this project.

**Comment by maxekman** [comment]
Hi, and thanks! Cool that you have made a ready to run container. What do you think about contributing it directly into this repo?

**Comment by Bekt** [comment]
That is fine with me as long as you guys can maintain it -- basically keep the `build.sh` file updated with the gliderlabs/logspout:master. I published this about a month ago and it seems to already have 1k downloads (not sure if real or bots), so that's another issue.

**Comment by maxekman** [comment]
Definitely! Would it be possible for you to make a PR with a docker folder containing the related files and append the contents of your readme to a new section (but maybe keep the badges at the top)? I'll add testing and code coverage soon to join your docker badges. 😊

Don't forget to credit yourself in the Docker section in the readme if you want!

**Comment by Bekt** [comment]
Just an update: I still want/have plans to do this, I just haven't had the time.

Just to make sure: do you want to host this on your DockerHub or would like to keep it on mine? I can give you/others access, I just don't any breaking changes for others (currently at 36k downloads). Would like to hear what would be the best for you guys since you're basically taking over.

**Comment by maxekman** [comment]
Hi, sorry for the delay. We can host it on our account. Maybe you can add a note on your repo that it is not maintained and point to the new location?
